docs/tool_jmap.md

```diff
@@ -69,4 +69,12 @@ The following tool limitations apply:
 - Displaying data from core dumps is not supported; use `jdmpview` instead.
 - Other options , such as `-F` (force a dump of an unresponsive process) can be accomplished using `kill -QUIT <pid>`.
 
+The tool uses the Attach API, and has the following limitations:
+
+- Displays information only for local processes that are owned by the current user, due to security considerations.
+- Displays information for OpenJ9 Java processes only
+- Does not show information for processes whose Attach API is disabled. :fontawesome-solid-pencil-alt:{: .note aria-hidden="true"} **Note:** The Attach API is disabled by default on z/OS.
+
+For more information about the Attach API, including how to enable and secure it, see [Support for the Java Attach API](https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.vm.80.doc/docs/attachapi.html).

```

Comment:
I just spotted that this link is wrong. Updating the link isn't part of the issue but it might as well be fixed here as the other tools topics have the same problem. The Attach API section is one that we moved to the OpenJ9 docs, so we shouldn't be pointing at the old SDK docs any more. Most of the OpenJ9 topics have been fixed but for some reason these tools topics haven't. Can you please change the link to `[Java Attach API](attachapi.md)` (same for tool_jcmd.md, tool_jps.md, tool_jstat.md, and tool_jstack.md too), thanks.